package-lock.json

```diff
@@ -3837,9 +3838,9 @@
 			"integrity": "sha512-x/rqGMdzj+fWZvCOYForTghzbtqPDZ5gPwaoNGHdgDfF2QA/XZbCBp4Moo5scrkAMPhB7z26XM/AaHuIJdgauA=="
 		},
 		"node_modules/@babel/runtime": {
-			"version": "7.24.8",
-			"resolved": "https://registry.npmjs.org/@babel/runtime/-/runtime-7.24.8.tgz",
-			"integrity": "sha512-5F7SDGs1T72ZczbRwbGO9lQi0NLjQxzl6i4lJxLxfW9U5UluCSyEJeniWvnhl3/euNiqQVbo8zruhsDfid0esA==",
+			"version": "7.25.7",
+			"resolved": "https://registry.npmjs.org/@babel/runtime/-/runtime-7.25.7.tgz",
+			"integrity": "sha512-FjoyLe754PMiYsFaN5C94ttGiOmBNYTf6pLr4xXHAT5uctHb092PBszndLDR5XA/jghQvn4n7JMHl7dmTgbm9w==",
```

Comment:
A nice followup would be to update the pinned Babel packages in the root `package.json`. For example, the `@babel/core` package is pinned to 7.24.3 there, but the dependencies like `@babel/runtime` are now at a newer version, because they are not pinned.